### PR TITLE
Localise new 8.3 auctionator release

### PR DESCRIPTION
**Pull request by plusmouse**
Design review. Left the shopping list stuff as that's being reworked.

Utilities.Message usage needs updating

### CODE REVIEW

File: Source/Locales/Main.lua [hunk 1]
```diff
@@ -0,0 +1,28 @@
+ATR_L = {}
+
+local function fixMissingTranslations(incomplete, locale)
+  if locale == "enUS" then
+    return
+  end
+  local enUS = AUCTIONATOR_LOCALES["enUS"]()
```
Comment (jrob8577) [inline Source/Locales/Main.lua:7]: whitespace please

File: Source/Locales/Main.lua [hunk 2]
```diff
@@ -0,0 +1,28 @@
+ATR_L = {}
+
+local function fixMissingTranslations(incomplete, locale)
```
Comment (jrob8577) [inline Source/Locales/Main.lua:3]: Will we need this when we use the curse tooling?

Comment (plusmouse) [reply Source/Locales/Main.lua]: We probably won't

Comment (plusmouse) [reply Source/Locales/Main.lua]: But it will remain useful for offline testing

**Pull request merged**